### ISSUE TITLE
Payment Button Currency - UX issue

### ISSUE THREAD

**Issue by bitcoinshirt** [btcpayserver/btcpayserver#253]
Payment button gives a bit way too much customization options in the "currency" field which may lead to errors and confusion.

In my opinion, it's better to have a drop down to avoid errors, then having users using different abbreviations. One can put USD, someone can but US Dollar, someone can put just $, not to mention other currencies. 

Alternatively, there can be a tooltip saying what kind of currency format is required.

To summarize, I propose to standardize and display currencies in a drop-down form, instead of leaving it to the user.

**Comment by bitcoinshirt** [comment]
In the end, it appears that this has not caused issues as I expected. So closing it for now.
